src/queryWizard/QueryWizard.js

```diff
@@ -12,6 +12,7 @@ export default withStateLens({ current: 0 })(
   observer(
     ({
       tree,
+      path,
```

Comment:
@daedalus28 regarding the `path` prop and roundtripping with other search formats -- my thinking, to still allow arbitrary numbers of `WizardStep`s (and avoid having to match some hardcoded collection of `stepN` keys to render step groups), is to have QueryWizard handle separate paths for:
- "steps", a group node whose children are step groups, where the first is titled **"Search for ${schemaLabel} by..."** in QueryWizard and the rest are titled **"And..."** (which we can show together in a non-editable box in the other modes), and
- "filters", the path to the editable group in the other modes (so, typically `root/criteria/filters`), which is rendered as the final **"Narrow Your Results"** step in QueryWizard

I can think of a few way to handle this, on the spectrum of simple/clear vs easy/flexible:

1. `stepsPath` and `filtersPath` props
2. `path` prop for the steps path, `filtersPath` (or `additionalFilters` or something) for the filters path
4. All three of `stepsPath`, `filtersPath`, and `path`; if the latter is used, it looks for `steps` and `filters` children on the node at that path

What do you think?